src/commands/hld/pipeline.ts

```diff
@@ -74,6 +66,15 @@ export const populateValues = (opts: ICommandOptions) => {
 
   opts.buildScriptUrl = opts.buildScriptUrl || BUILD_SCRIPT_URL;
 
+  if (!opts.hldUrl || !opts.manifestUrl) {
```

Comment:
@yradsmikham this entire code block is contained. We don't reuse any of the const variables. Can we just encapsulate the entire block in a function?

As I look at the rest of the `populateValues` function there is refactoring that we can do outside of this PR

Reply:
Sure, will do.